app/views/patterns/task-list/index.html

```diff
@@ -0,0 +1,33 @@
+{% extends "layout.html" %}
+
+{% block page_title %}
+  Task list variations
+{% endblock %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+  <h1 class="heading-large">
+    Task list variants
+  </h1>
+
+  <p>
+    <a href="/patterns/task-list/table">Example using a table</a><br>
+    This pattern uses a table, in a similar way to the check-your-answers pattern.
+  </p>
+
+  <p>
+    <a href="/patterns/task-list/list">Example using Headings and lists</a><br>
+    This pattern uses heading levels <code class="code">&lt;h1&gt;</code> to <code class="code">&lt;h3&gt;</code> for the task section headings, and an ordered list for the task items.
+  </p>
+
+  <p>
+    <a href="/patterns/task-list/list-hide-section-numbers">Example using section</a>
```

Comment:
This link doesn't work. I just fixed it in 83b0e1a977694da06066e7add44044d41b4c381c.